ts/theme/components/ScreenHeader.ts

```diff
@@ -0,0 +1,15 @@
+import { Theme } from "../types";
+import variables from "../variables";
+
+export default (): Theme => ({
+  "NativeBase.ViewNB": {
+    "NativeBase.ViewNB": {
+      // margin to align icon to the baseline of the title
+      marginBottom: variables.fontSize6 / 3
```

Comment:
Is this supposed to be 1/3 of `fontSize6` (i.e. if I change `fontSize6`, does this margin have to change accordingly)?

Reply:
Yes, in order to keep the icon aligned to the baseline of the title

Reply:
But if I change `fontSize6` to, say, `10 * fontSizeBase`, the result I get is the following:
<img width="371" alt="screen shot 2018-06-27 at 12 54 04 pm" src="https://user-images.githubusercontent.com/107715/41969923-50296d7e-7a09-11e8-99f3-6201f4d97974.png">

I don't see how those two values (`fontSize*` and that bottom margin) can be related, so if you could elaborate on the reasoning behind it, it would help me understand it.

Reply:
My fault, I thought that the `<H1 />` font-size was setted by `fontSize6` theme variable. Fixed.